src/Razor/benchmarks/Microsoft.AspNetCore.Razor.Microbenchmarks/ProjectSystem/ProjectSnapshotManagerBenchmarkBase.cs

```diff
@@ -93,7 +93,7 @@ private static IReadOnlyList<TagHelperDescriptor> ReadTagHelpers(string filePath
 
         private class TestProjectSnapshotManagerDispatcher : ProjectSnapshotManagerDispatcher
         {
-            protected override bool IsDispatcherThread => true;
+            public override bool IsDispatcherThread => true;
```

Comment:
I juuust changed this to `protected` in my previous rename PR, but am already having to change it back to `public`. 😢